Makefile

```diff
@@ -332,6 +345,10 @@ test: $(SETUP_ENVTEST) manifests test-exp-etcdrestore ## Run all generators and
 test-exp-etcdrestore: $(SETUP_ENVTEST) ## Run tests for experimental etcdrestore API.
 	cd $(EXP_ETCDRESTORE_DIR); KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" go test ./... $(TEST_ARGS)
 
+.PHONY: test-exp-clusterclass
```

Comment:
We used to have similar code for etcdrestore however we removed it recently in https://github.com/rancher/turtles/pull/897/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L187 and made it to be built as part of the turtles image (same goes for docker build commands below). Do you want to align on that in this PR?

Reply:
I'd say we do it in the near future when we polish this functionality and put it behind a feature gate. For now I think it's cleaner/safer to maintain a separate process for image building. What do you think?

Reply:
Fine with me, it was more of JFYI comment here.